app/views/SidebarView/index.js

```diff
@@ -180,21 +183,30 @@ class Sidebar extends Component {
 				<SidebarItem
 					text={I18n.t('Chats')}
 					left={<CustomIcon name='message' size={20} color={themes[theme].titleText} />}
-					onPress={() => this.sidebarNavigate('ChatsStackNavigator')}
+					onPress={() => {
+						logEvent(events.NAVIGATE_TO_ROOMSLIST);
```

Comment:
All of these calls should be done inside of the function.
It's odd to add rules inside of the render function, since we already have a function to do it.

Reply:
I placed a logic to log the event inside `sidebarNavigate()` based on the passed route, but in this component `Navigation.navigate()` is being used twice, should I place the same logic there or change to use `sidebarNavigate()`?

Reply:
Check if it works and try to remove the other one.

Reply:
It works fine using `this.sidebarNavigate()` instead of `Navigation.navigate()`